pkg/controller/pxc/controller.go

```diff
@@ -239,10 +239,10 @@ func (r *ReconcilePerconaXtraDBCluster) Reconcile(ctx context.Context, request r
 				sfs = statefulset.NewProxy(o)
 				// deletePVC is always true on this stage
 				// because we never reach this point without finalizers
-				err = r.deleteStatefulSet(o, sfs, true)
+				err = r.deleteStatefulSet(o, sfs, true, true)
```

Comment:
We should not remove secrets in the case of delete-proxysql-pvc finalizer. We still have DB in PXC's PVCs

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1380/commits/9e145182b349be13bc0fe4ae3cb3aed6c196546f